spec/models/ability_spec.rb

```diff
@@ -2,42 +2,48 @@
 require 'cancan/matchers'
 
 RSpec.describe Ability do
+  let(:staff) { FactoryGirl.create(:user) }
+  let(:admin) { FactoryGirl.create(:admin) }
+  let(:admin_role) { Role.create name: "admin" }
+  before do
+    admin_role.users << admin
+    admin_role.save
+  end
```

Comment:
If FactoryGirl has a factory defined `:admin`, shouldn't it already be doing this? I see in fact the only difference in the admin factory is the email! 

I think the admin factory should probably be fixed to just do this in the factory. Probably with a find_or_create so it's nice and lazy. 

Otherwise, I'd get rid of the admin factory altogether. And make the user factory use FG's sequence feature, so every one doesn't have the same email, and you can create multiple. 

What do you think?  Just suggestions, we can merge as is depending on your thoughts.